pkgs/tools/networking/assh/default.nix

```
@@ -15,6 +20,10 @@ buildGoModule rec {
 
   doCheck = false;
 
+  preBuild = ''
+    buildFlagsArray+=("-ldflags" "-s -w -X moul.io/assh/v2/pkg/version.Version=${version}")
+  '';
```

```suggestion
  buildFlagsArray = [ "-ldflags" "-s -w -X moul.io/assh/v2/pkg/version.Version=${version}" ];
```
I think you can do this instead

---

This is fine. See https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-flags-array.md for the reasoning behind this.